src/server/routes/queueDelete.ts

```diff
@@ -0,0 +1,188 @@
+import { Request, Response } from 'express';
+import { Router } from 'express';
+import { checkSchema, Schema } from 'express-validator';
+import { dynamoClient, readClient, writeClient } from '../../database/client';
+import config from '../../config';
+import { sqs } from '../aws/sqs';
+import {
+  SendMessageBatchCommand,
+  SendMessageBatchRequestEntry,
+} from '@aws-sdk/client-sqs';
+import { nanoid } from 'nanoid';
+import * as Sentry from '@sentry/node';
+import { validate } from './validator';
+import { HighlightsDataService } from '../../dataservices/highlights';
+import { NotesDataService } from '../../dataservices/notes';
+import { failCallback, successCallback } from './helper';
+
+export type SqsMessage = {
+  userId: number;
+  email: string;
+  isPremium: boolean;
+  annotationIds: number[];
+  traceId: string;
+};
+
+const router = Router();
+
+const queueDeleteSchema: Schema = {
+  traceId: {
+    in: ['body'],
+    optional: true,
+    isString: true,
+    notEmpty: true,
+  },
+  userId: {
+    in: ['body'],
+    errorMessage: 'Must provide valid userId',
+    isInt: true,
+    toInt: true,
+  },
+  isPremium: {
+    in: ['body'],
+    errorMessage: 'Must provide valid isPremium (true | false)',
+    isBoolean: true,
+  },
+};
+
+router.post(
```

Comment:
router is added to main.ts in https://github.com/Pocket/annotations-api/pull/266